src/core/shape/2d_primitives.js

```diff
@@ -375,6 +375,17 @@ p5.prototype._renderEllipse = function(x, y, w, h, detailX) {
  * </code>
  * </div>
  *
+ * <div>
+ * <code>
+ * createCanvas(100, 100, WEBGL);
+ * strokeWeight(7);
+ * line(0, 0, 0, 30, 0, 0);
+ * line(0, 0, 0, 0, 30, 0);
+ * line(-30, -30, -30, 30, 30, 30);
+ * describe('two lines mark the flat x and y axes while a third line spans diagonally through the third dimension');
+ * </code>
+ * </div>
```

Comment:
The other code examples only used the x and y parameters for [line()](https://p5js.org/reference/#/p5/line), so I added one using the z parameters for WEBGL mode. 

I figure it's a good goal to show all the possible parameters in the code examples, which would mean having a 3D example for the 2D primitives.

Having these scaffolded examples might naturally help users transition to WEBGL. Eg. they might know the WEBGL coordinate system is different just from their time browsing 2D primitives examples.

Reply:
Agree, it's nice to see the webGL versions with the updated signatures! If we don't think it's too much code I think it could be nice to color these lines in rgb (xyz) gizmo fashion like so https://editor.p5js.org/aferriss/sketches/qCsZ08lJ8